internal/cmdliveinit/cmdliveinit_test.go

```diff
@@ -223,22 +241,29 @@ func TestCmd_Run(t *testing.T) {
 				return
 			}
 
-			// Otherwise, validate the kptfile values
+			// Otherwise, validate the kptfile and resourcegroup values
 			assert.NoError(t, err)
 			kf, err := pkg.ReadKptfile(w.WorkspaceDirectory)
 			assert.NoError(t, err)
-			if !assert.NotNil(t, kf.Inventory) {
+			if !assert.Nil(t, kf.Inventory) {
+				t.FailNow()
+			}
+			assert.NoError(t, err)
+			rg, err := pkg.ReadRGFile(w.WorkspaceDirectory, rgfilename)
+			assert.NoError(t, err)
+			if !assert.NotNil(t, rg) {
 				t.FailNow()
 			}
-			actualInv := *kf.Inventory
+
+			actualInv := kptfilev1.Inventory{
```

Comment:
This is a bit confusing. If we write the inventory information into an RG resource, why put it back into the Kptfile structure here?